examples/test/BasicComponents_test.js

```diff
@@ -4,14 +4,15 @@ import { config } from "./utils/config";
 import { loginUsingLoginForm } from "./utils/helpers";
 
 async function checkRenderChart(selector, t) {
-    const loading = Selector(".s-loading");
-    const chart = Selector(selector);
+    const loading = await Selector(".s-loading");
+    const chart = await Selector(selector);
```

Comment:
Why is the selector awaited? It does not need to be imho. https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/creating-selectors.html

Reply:
@xMort I forgot to remove it. Solved.